runtime/compiler/typst.vim

```diff
@@ -0,0 +1,34 @@
+" Vim compiler file
+" Compiler: typst
+
+if exists('current_compiler')
+    finish
+endif
+let current_compiler = get(g:, 'typst_cmd', 'typst')
+
+let s:save_cpo = &cpo
+set cpo&vim
+
+if exists(":CompilerSet") != 2
+    command -nargs=* CompilerSet setlocal <args>
+endif
```

Comment:
We got rid of the fallback for the `:CompilerSet` command back in 408281e16a36c15eed10fbf0406fa8ab159fc4bf